tests/hypermodel/test_preprocessor.py

```diff
@@ -113,3 +120,36 @@ def map_func(x):
     for _ in new_dataset:
         pass
     assert isinstance(new_dataset, tf.data.Dataset)
+
+
+def test_lgbm(tmp_dir):
+    x_train = np.random.rand(11, 32)
+    y_train = np.array([[1, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+                        [1, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+                        [0, 1, 0, 0, 0, 0, 0, 0, 0, 0],
+                        [0, 0, 0, 0, 1, 0, 0, 0, 0, 0],
+                        [0, 0, 0, 0, 0, 0, 0, 1, 0, 0],
+                        [0, 0, 0, 0, 0, 1, 0, 0, 0, 0],
+                        [0, 0, 0, 0, 0, 0, 0, 0, 0, 1],
+                        [0, 0, 0, 0, 0, 0, 0, 1, 0, 0],
+                        [1, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+                        [0, 0, 0, 1, 0, 0, 0, 0, 0, 0],
+                        [0, 0, 0, 1, 0, 0, 0, 0, 0, 0]])
+
+    input_node = ak.Input()
+    output_node = input_node
+    output_node = ak.hypermodel.preprocessor.LgbmClassifier()(output_node)
+    output_node = ak.IdentityBlock()(output_node)
```

Comment:
This is a strange setup. We should not require users to use objects that don't have a clear purpose like IdentityBlock or EmptyHead.